geoalchemy2/functions.py

```diff
@@ -75,6 +75,9 @@ def __init__(cls, clsname, bases, clsdict):
 
 
 class TableRowElement(ColumnElement):
+
+    inherit_cache = True
+
```

Comment:
this one also looks like inherit_cache=False

if I have:

t1 = TableRowElement(select(Foo))
t2 = TableRowElement(select(Bar))

do t1 / t2 produce identical SQL ?   keep in mind, inherit cache means, t1 and t2 will have the same cache key.  t1 will generate SQL, t2 will not.    seems unlikely here

Reply:
Indeed, I think they could produce different SQL so let's set it to False.